### DISCUSSION TITLE
dial tcp ***:***: i/o timeout

### DISCUSSION THREAD

**Discussion by nugroh03** [community/community#121274] [Actions]
### Select Topic Area

Question

### Body

i use github action for update my program in vps, but today i have error dial tcp.  i not change anything on my vps.

**Comment by grandmas-favorite** [comment]
Hi @nugroh03!

Would you be willing to provide a little more insight into the issue, perhaps with screenshots and expanded error information? I believe this would help myself and other Community members understand the issue better!

**Reply by nugroh03** [reply]
Solve. Problem on provider vps. Thank you for your reponse.
